csm-extensions/services/dev-postgres/scripts/build-db.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-
-pushd ${SIDECAR_EXTENSION_ROOT}/db
- docker build -t ${SIDECAR_EXTENSION_SVC_IMAGE_NAME}:${SIDECAR_EXTENSION_SVC_IMAGE_TAG} .
-popd
\ No newline at end of file
+topdir=$(dirname $(dirname "$0"))
```

Comment:
So this is just `${SIDECAR_EXTENSION_ROOT}`, then? :)

Reply:
Ah, I guess so. Will fix. Note, mysql has the identical construct.

Reply:
Done with commit 68039d16c4d0987ec34b692adc51597e9755179a